core-web/src/main/java/org/silverpeas/core/webapi/profile/UserProfileResource.java

```diff
@@ -241,20 +255,20 @@ public Response getApplicationUsers(
       @QueryParam("page") String page,
       @QueryParam("userStatesToExclude") Set<UserState> userStatesToExclude) {
     String[] roleNames = isDefined(roles) ? roles.split(",") : null;
-    String domainId = null;
+    List<String> domainIds = new ArrayList<String>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS2293)